### PR TITLE
Join by QR-Code

### PR DESCRIPTION
**Pull request by DonkeyCo**
This feature is still in progress, but I'm already finished with part one and would appreciate a short review, before I sit down to implement the joining endpoint.

Currently there is the /fridge/management/<fid>/qr-code endpoint, which creates a short dynamic link with the help of Firebase. It also creates an Accesstoken, which is later used for authorizing any user joining the fridge. Currently each link has a validity of 12 hours.

Also had to change some stuff in the compose files, as we now use a .env file, which caused problems with the automated builds. We have to talk about where to place them in the build pipeline as well, as they store crucial data such as API keys.

### CODE REVIEW

**Comment by DonkeyCo** [comment]
Added join endpoint, so the implementation is ready. Would appreciate a review and your feedback to the code.

File: Fridgify_Backend/views/fridge/management/join_fridge.py [hunk 1]
```diff
@@ -1,5 +1,64 @@
...
+@permission_classes([IsAuthenticated])
+def join_view(request):
+    token = request.GET.get("token")
+    if token is None:
+        raise ValidationError(detail="Missing token argument")
+
+    try:
+        token_obj = Accesstokens.objects.filter(provider__name="Fridgify-Join", accesstoken=token).get()
+    except Accesstokens.DoesNotExist:
+        raise NotFound(detail="Join Link not found")
```
Comment (JoschuaGoetz) [inline Fridgify_Backend/views/fridge/management/join_fridge.py:49]: Little typo

Comment (JoschuaGoetz) [reply Fridgify_Backend/views/fridge/management/join_fridge.py:49]: well should be lower case.

Comment (DonkeyCo) [reply Fridgify_Backend/views/fridge/management/join_fridge.py]: Should be fixed now

**Pull request merged**